core/src/main/java/com/datastrato/gravitino/Configs.java

```diff
@@ -254,8 +254,11 @@ public interface Configs {
           .doc("The admins of Gravitino service")
           .version(ConfigConstants.VERSION_0_5_0)
           .stringConf()
-          .checkValue(StringUtils::isNotBlank, ConfigConstants.NOT_BLANK_ERROR_MSG)
           .toSequence()
+          .checkValue(
+              valueList ->
+                  valueList != null && valueList.stream().allMatch(StringUtils::isNotBlank),
+              ConfigConstants.NOT_BLANK_ERROR_MSG)
```

Comment:
Why the ordering matters? I think it is not just simply changing the ordering to fix the issue. We can misuse it if the implementation is strongly relying on the ordering, right?

Reply:
I modify the implement to fix the issue.
1. If we call this method before `toSequence`, it will check every element.
2. If we call this method after `toSequeue`, it will check the list.